content/howtos/_index.md

```diff
@@ -14,3 +14,6 @@ This section includes LocalAI end-to-end examples, tutorial and how-tos curated
 - [Model Setup by Downloading Model]({{%relref "howtos/easy-model-import-downloaded" %}})
 - [Making requests via OpenAi API]({{%relref "howtos/easy-request-openai" %}})
 - [Making requests via Curl]({{%relref "howtos/easy-request-curl" %}})
+
+This section includes Full auto setup, install, and use of LocalAI by [lunamidori5](https://github.com/lunamidori5).
```

Comment:
this sounds a slightly repetition of https://github.com/go-skynet/localai-website/blob/c34e3fb1816fa7f8fa9058e72c9f78aaa88e7451/content/howtos/_index.md?plain=1#L9

Reply:
I can remove it if you like @mudler